Sources/focus-window/FocusWindow.swift

```diff
@@ -1,67 +1,77 @@
 import Cocoa
-import Foundation
 
-class FocusWindow {
-  static func focusWindow(_ windowNum: String) -> Bool {
+final class FocusWindow {
+  static func focusWindow(windowNumber: Int) {
     if !isTrusted() {
       print("Missing Accessibility Permissions", to: .standardError)
       exit(1)
     }
 
-    if let windowNumber = Int(windowNum) {
-      let options = CGWindowListOption(arrayLiteral: CGWindowListOption.optionOnScreenOnly)
-      let windowList = (CGWindowListCopyWindowInfo(options, kCGNullWindowID) as! NSArray).map { $0 as! NSDictionary }
+    let options = CGWindowListOption(arrayLiteral: .optionOnScreenOnly)
+    let windowList = CGWindowListCopyWindowInfo(options, kCGNullWindowID) as! [[String: Any]]
```

Comment:
The Swift bridging is pretty good, so you can just force cast deeply to what you want.